cmd/init.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+
+	"github.com/docker/docker/pkg/term"
+	docker "github.com/fsouza/go-dockerclient"
+	"log"
+	"os"
+)
+
+func newInitCommand(opts *dobiOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Run the remove action for all resources",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runInit(opts)
+		},
+	}
+	flags := cmd.Flags()
+	flags.StringVarP(&opts.cookie, "cookie", "c", "https://github.com/cescoferraro/go-dobi-cutter", "Url to")
+	return cmd
+}
+
+func runInit(opts *dobiOptions) error {
+	client, err := buildClient()
+	if err != nil {
+		return fmt.Errorf("Failed to create client: %s", err)
+	}
+	container, err := startCookieCutterContainer(client)
+	if err != nil {
+		return fmt.Errorf("Failed to start container %s", err)
+	}
+	err = attachToCookieContainer(client, container)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func startCookieCutterContainer(client docker.Client) (docker.Container, error) {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return docker.Container{}, err
+	}
+	dockeropts := docker.CreateContainerOptions{
```

Comment:
It's kind of unfortunate that we now have this "container run" code in two places, one for the job task and one for init.

I'm wondering if we could re-use `dobi`s ability to run containers to make this work.

If we added a `--url` flag (similar to `-f`) to point `dobi` at a remote `dobi.yaml`, we could do something like this:

```
dobi --url https://raw.githubusercontent.com/cescoferraro/go-dobi-cutter/master/dobi.yaml
```

This solution works really well with [the idea](https://github.com/dnephin/dobi/issues/22#issuecomment-257202850) for allowing`include` to also be remote urls. The code should be about the same, so can be mostly re-used.

It makes it easy for anyone to create their own init, or other bootstrapping process, it doesn't even have to use cookiecutter.

The only issue I see is that the github urls are a bit long, but that can be solved pretty easily with either url shortners, or with a better host for the `dobi.yaml` files.